src/controllers/destination.ts

```diff
@@ -94,10 +93,7 @@ export class DestinationController {
 
   public static async destinationTransformAtRouter(ctx: Context) {
     const startTime = new Date();
-    logger.debug(
-      'Native(Router-Transform):: Requst to transformer::',
-      JSON.stringify(ctx.request.body),
-    );
+    logger.debug('Native(Router-Transform):: Requst to transformer::', ctx.request.body);
```

Comment:
Do we need this log ?

Reply:
this is present in existing code ... might be useful in tracking from beginning of calls and ctx body

Reply:
Ok.